transport/grpc/handler_interface.go

```diff
@@ -0,0 +1,20 @@
+package grpc
+
+import (
+	"golang.org/x/net/context"
+	"google.golang.org/grpc"
+)
+
+/*
+This needs to be in a separate file because the gRPC interface does not
+follow golint rules, so we put it in this file so we can ignore it when
+running go lint
+*/
+func (h handler) Handle(
```

Comment:
nit: add an empty line between the function and the comment since this comment
isn't documenting the function.